samples/charts/data-chart/scatter-series/Pages/DataChartScatterSeries.razor

```diff
@@ -0,0 +1,218 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+
+    <div class="igOptions-horizontal">
+        <span class="igOptions-label">Type of Scatter Series: </span>
+                    <select value=@SeriesType
+                        @onchange="OnSeriesTypeChanged">
+                        <option>Bubble</option>
+                        <option>Point</option>
+                        <option>Line</option>
+                        <option>Spline</option>
+                    </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (data1 != null && data2 != null)
```

Comment:
there should be 6 @if (SeriesType == "ScatterBubbleSeries" ) conditions each with one type of series (see StackedSeries sample):
ScatterAreaSeries, ScatterBubbleSeries, Scatter(Point)Series, ScatterLineSeries, ScatterSplineSeries, ScatterContourSeries